src/main/java/exnihilocreatio/client/renderers/RenderAutoSifter.java

```diff
@@ -40,7 +40,7 @@ public void render(TileAutoSifter te, double x, double y, double z, float partia
             }
         }
 
-        if (te.connectionPieces != null && ModConfig.client.clientFancyAutoSieveAnimations) {
```

Comment:
Why remove the null check?

Reply:
My IDE reported that te.connectionPieces != null was always true, so it felt like a pointless check